vms/platformvm/block/builder/helpers_test.go

```diff
@@ -164,8 +173,8 @@ func addSubnet(t *testing.T, env *environment) {
 			ts.Keys[1].PublicKey().Address(),
 			ts.Keys[2].PublicKey().Address(),
 		},
-		[]*secp256k1.PrivateKey{ts.Keys[0]},
-		ts.Keys[0].PublicKey().Address(),
+		[]*secp256k1.PrivateKey{ts.Keys[4]},
```

Comment:
we used to use key 0 to fund a subnet creation. This messes up the key 0 balance (it reduces its initial balanche `ts.Balance` of the subnet creation fee) which can be annoying in  some UTs where we do balance   comparison.
In such tests we rarely use key 4 instead, so it's easier to use key 4 here to fund the subnet creation and keep balance checks simpler